public/js/like.js

```diff
@@ -16,8 +17,10 @@ $(document).ready(function() {
 			likeEpisode()
 
 			like_count = Number(like_count) + 1;
+			favoriteCount = Number(favoriteCount) + 1
```

Comment:
Missing semicolon.<br>'favoriteCount' is not defined.<br>Too many errors. (17% scanned).